org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -93,34 +117,27 @@ def installed() {
 def updated() {
 	log.debug "Updated with settings: ${settings}"
 	unsubscribe()
-	initialize()
```

Comment:
Yes. since we donnot want to reset variables like connectionId and endpointURL.

---
In reply to: [100166841](https://github.com/openT2T/translators/pull/152#discussion_r100166841) [](ancestors = 100166841)